src/pages/home.tsx

```diff
@@ -1,9 +1,108 @@
-import React from "react";
+import './home.css';
+import React from 'react';
+import { ReactComponent as PageLinksIcon } from '../assets/arrow.svg';
+import { ReactComponent as CreateHackathonIcon } from '../assets/createHackathonIcon.svg';
+import { ReactComponent as HackathonAnalyticsIcon } from '../assets/hackathonAnalyticsIcon.svg';
+import { Link } from 'react-router-dom';
 
 export default function Home() {
-  return (
-    <div>
-      <h1>HOME</h1>
-    </div>
-  );
+	// TODO: This user name should be generic.
+	let userName: string = 'John';
+
+	return (
+		<div id="page">
+			<div id="page-header">
+				<div id="page-header-main">
+					<div id="page-header-main-generic-profile-icon" />
+					<h1 id="page-header-main-text">Welcome, {userName}.</h1>
+				</div>
+
+				<Link to="/analytics">
+					<button id="page-header-button">
+						<HackathonAnalyticsIcon id="page-header-button-icon" />
+						<body id="page-header-button-text">
+							Hackathon Analytics
+						</body>
+					</button>
+				</Link>
+			</div>
+
+			<h1 id="page-subheader-1-text">Upcoming Hackathon</h1>
+
+			<div id="page-links">
+				<h1 id="page-links-header">Fall 2022</h1>
+
+				<div id="page-links-grid">
```

Comment:
These links don't play well when we make the page smaller. Text overflows outside of the links and the boxes are still fixed 3 per row. It would be helpful to make this use breakpoints and flex so that we fit more in a row at larger screen sizes and fewer (down to one) per row at smaller screen sizes.